index.html

```diff
@@ -54,5 +55,148 @@ <h1>I’m Fadahunsi Glad to see you!</h1>
             </ul>
         </div>
     </section>
+    <main>
+        <section id="myWork" class="my-works">
+            <div class="work-item">
+                <img alt="work-item-img" src="./img/work/work-1-cover.png"/>
+              <h3 class="work-title">Tonic</h3>
+              <div class="work-subtitle">
+                <span>Canopy</span>
+                <span>Back End Dev</span>
+                <span>2015</span>
+              </div>
+              <p class="work-details">
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+              </p>
+              <ul class="work-technologies">
+                <li class="technology-tag">html</li>
+                <li class="technology-tag">css</li>
+                <li class="technology-tag">javascript</li>
+              </ul>
+              <button class="view-work-btn">See Project</button>
+            </div>
+            <div class="work-item">
+                <img alt="work-item-img" src="./img/work/work-2-cover.png"/>
+              <h3 class="work-title">Multi-Post Stories</h3>
+              <div class="work-subtitle">
+                <span>Canopy</span>
+                <span>Back End Dev</span>
+                <span>2015</span>
+              </div>
+              <p class="work-details">
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+              </p>
+              <ul class="work-technologies">
+                <li class="technology-tag">html</li>
+                <li class="technology-tag">css</li>
+                <li class="technology-tag">javascript</li>
+              </ul>
+              <button class="view-work-btn">See Project</button>
+            </div>
+            <div class="work-item">
+              <img alt="work-item-img" src="./img/work/work-3-cover.png"/>
+              <h3 class="work-title">Tonic</h3>
+              <div class="work-subtitle">
+                <span>Canopy</span>
+                <span>Back End Dev</span>
+                <span>2015</span>
+              </div>
+              <p class="work-details">
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+              </p>
+              <ul class="work-technologies">
+                <li class="technology-tag">html</li>
+                <li class="technology-tag">css</li>
+                <li class="technology-tag">javascript</li>
+              </ul>
+              <button class="view-work-btn">See Project</button>
```

Comment:
- [ ] please try to add the button effect on pressed  in order to match the figma design 
![image](https://user-images.githubusercontent.com/61790587/170545012-4f1b5f82-0435-4c6d-b7e3-d2d51a4633fc.png)